src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.tsx

```diff
@@ -34,15 +37,26 @@ type Props = {
 const DefaultTranscriptslist = (props: Props) => {
   const { gene } = props;
   const sortedTranscripts = defaultSort(gene.transcripts);
+  const [isFilterOpen, setFilterOpen] = useState(false);
+
+  const toggleFilter = () => { isFilterOpen ? setFilterOpen(false) : setFilterOpen(true); }
 
   return (
     <div>
       <div className={styles.header}>
+        {isFilterOpen && <GeneFilter toggleFilter={toggleFilter} />}
         <div className={styles.row}>
-          <div className={styles.left}>Filter & sort</div>
-          <div className={styles.middle}></div>
+          <div className={isFilterOpen ?  styles.hidden : styles.filterLabel}>
+            Filter & sort
+            <button onClick={toggleFilter} className={styles.expandBtn}>
+              <img
+                src={chevronDownIcon}
+                alt={'collapse'}
+              />
+            </button>            
+          </div>          
           <div className={styles.right}>Transcript ID</div>
-        </div>
+        </div>       
```

Comment:
We don't want trailing empty space. Is it possible that `prettier` is not working for you? I thought it would fix these things automatically.

Reply:
It wasn't installed. I just realised i had to uninstall and install vs code again to make it work with wsl properly. It did fix the issue with wsl but i guess i forgot to enable some settings and extensions :)